.devcontainer/devcontainer.json

```
@@ -0,0 +1,21 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/universal
+{
+  "name": "Default Linux Universal",
```

```suggestion
  "name": "scientific-python-cookie[dev]",
```

Is there anything special with `[dev]`?

---

Not really, I use that sometimes to indicate that it setups all the dev tooling and not just a user environment.